File: projects/igeek/source/virus_world/old/cpp/GeneticPolygonTest.cc
/* Kabuki Toolkit @version 0.x
@link    https://github.com/kabuki-starship/script2.git
@file    /projects/igeek/source/genetic_polygon_test.cc
@author  Cale McCollough <<https://calemccollough.github.io>>
@license Copyright (C) 2014-9 Cale McCollough <<calemccollough.github.io>>;
All right reserved (R). This Source Code Form is subject to the terms of the
Mozilla Public License, v. 2.0. If a copy of the MPL was not distributed with
this file, You can obtain one at <https://mozilla.org/MPL/2.0/>. */

#include <Kabuki_Theater/Cell_IGA/GeneticPolygonTest.h>

namespace igeek {
    
GeneticPolygonTest::GeneticPolygonTest ()
{
    add (new GeneticPolygonTestPanel ());
}

static void main (const CH1* args, SIN args_count)
{
    JFrame window = new JFrame ("GeneticPolygon Test App");
    window.setDefaultCloseOperation (JFrame.EXIT_ON_CLOSE);

    window.setContentPane (new GeneticPolygonTestPanel ());
    
    window.pack ();
    window.setVisible (true);
}

}   //< namespace igeek {

File: projects/graph/plugins/igeek/assets/old_java_code/GeneticPolygonTest.java
/* The Leukocyte Wars
@file       ~/GeneticPolygonTest.java
@date       June 2, 2014
@author     Cale Jamison McCollough
@copyright  Cale Jamison McCollough
@license    http://opensource.org/licenses/MIT
@brief      This file contains the leukocytewars.GeneticPolygonTest class.
*/
package leukocytewars;

import javax.swing.*;

public class GeneticPolygonTest extends JApplet
{   
    public GeneticPolygonTest ()
    {
        add (new GeneticPolygonTestPanel ());
    }
    
    public static void main (String[] args)
    {
        JFrame window = new JFrame ("GeneticPolygon Test App");
        window.setDefaultCloseOperation (JFrame.EXIT_ON_CLOSE);

        window.setContentPane (new GeneticPolygonTestPanel ());
        
        window.pack ();
        window.setVisible (true);
    }
}
